src/com/dabomstew/pkrandom/constants/GlobalConstants.java

```diff
@@ -39,125 +39,73 @@
 
     public static final boolean[] bannedRandomMoves = new boolean[796], bannedForDamagingMove = new boolean[796];
```

Comment:
If you *do* add all the moves, this should be changed. We set it to 796 because that was the last move in SwSh at the time; they've since gone up to 826.

Reply:
Done. (827 because we are using 1-based indexes, yeah?)

Reply:
That's correct (and 796 is currently wrong). We need to *index* up to 826, so we need an 827-element array.